bundles/org.openhab.ui/src/main/java/org/openhab/ui/internal/UIService.java

```diff
@@ -39,7 +39,7 @@
  *
  * @author Yannick Schaus - Initial contribution
  */
-@Component(immediate = true, name = "org.openhab.ui")
+@Component(immediate = true, name = "org.openhab.ui", property = { "httpContext.id:String=oh-ui-http-ctx" })
```

Comment:
Oh, and thanks for taking care of this 🙂

Reply:
It will logs warnings about them being null if they are missing. :wink:

Reply:
yes I've seen them, and have been meaning to investigate for a while, but you know how it is ;)

Reply:
Uhm, what is that about?

Reply:
@kaikreuzer currently you get an ugly warning about null http context ids like this:
```
[WARN ] [d.internal.element.HttpContextElement] - Registered http context [null] did not contain a valid http context id
```

Reply:
Yes we also use it on the [SmartHomeHttpContext](https://github.com/openhab/openhab-core/blob/ac1f4080e3896d0a9b44cef3e5ca74efa58273ed/bundles/org.openhab.core.io.http/src/main/java/org/openhab/core/io/http/internal/SmartHomeHttpContext.java#L45-L47). 

I spent hours to get the default UI properly working with the Demo App on / because the default Whiteboard app always kept showing up, also after adding configurations to disable it. They removed it from v1.0.8 because it was causing issues for many users. In an attempt to get it fixed I also investigated this warning. :wink: